src/purchases.ts

```diff
@@ -187,13 +196,14 @@ export default class Purchases {
      * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults suite, otherwise it will use standardUserDefaults.
      * Default is null, which will make the SDK use standardUserDefaults.
      */
-    public static setup(
-        apiKey: string,
-        appUserID?: string | null,
-        observerMode: boolean = false,
-        userDefaultsSuiteName?: string,
-        usesStoreKit2IfAvailable: boolean = false
-    ): void {
+    public static setup({
```

Comment:
I've renamed this to `configure` in #374.

Reply:
Note this version uses a `PurchasesConfiguration` object as a single parameter, I am not a JS expert but I think this is a common practice when the functions have a lot of optional arguments